server/Models/snapshotModel.ts

```diff
@@ -0,0 +1,32 @@
+const myURI: String =
+  'mongodb+srv://codesmithbv:ax6POS3lHiGRGPfO@kalecluster.3wl6slc.mongodb.net/';
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+try {
+  mongoose.connect(myURI);
+  console.log('connected to mongodb');
+} catch (error) {
+  console.log(error);
+}
+
+const snapshotSchema = new Schema({
+  podName: String,
+  date: Date,
```

Comment:
suggestion! should we make the date to default to the current time when it's not provided?

Reply:
updated to make date default